docs/guides/developer/.generate-list-of-committers.sh

```diff
@@ -3,23 +3,28 @@
 set -eu
 
 # https://github.com/settings/tokens
-USER=<username>
-TOKEN=<personal-github-access-token>
+#GITHUB_USER=<username>
+#GITHUB_PA_TOKEN=<personal-github-access-token>
 
-echo 'List of Committers'
-echo '------------------'
-echo
-echo '<ul>'
+OUT=docs/participate/list-of-committers.md.include
+
+echo 'List of Committers' > "${OUT}"
+echo '------------------' >> "${OUT}"
+echo >> "${OUT}"
+echo 'The current list of committers in aplhabetical order:' >> "${OUT}"
+echo >> "${OUT}"
+echo '<ul>' >> "${OUT}"
 
 LINK='<a href=" + .html_url + ">'
 IMG='<img style=\"width: 40px; margin: 0\" src=" + .avatar_url + " /> "'
-curl -s -u "$USER:$TOKEN" \
+curl -s -u "$GITHUB_USER:$GITHUB_PA_TOKEN" \
```

Comment:
FTR, this does not work at all for me - the `curl` call always 404s (not 401, not 403).  Tested inside via the `gh` commandline tool, script works as expected.